### ISSUE TITLE
Name Change Suggestion

### ISSUE THREAD

**Issue by KevinBlue18** [maforget/ComicRack_AmazonScrapper#1]
Shouldn't the name be _Scraper_ and _Scrapper_ because a "scrapper" is the noun form for "scrap" which means to throw something away while "scrape" means to copy from a web page using a computer program?

**Comment by maforget** [comment]
You are right, but at this point renaming it would break any links that point to this github page.
